content/docs/dataform-web/guides/git-access-tokens.md

```diff
@@ -2,18 +2,48 @@
 title: Git access tokens
 ---
 
-Git access tokens are user to authenticate access to Dataform projects attached to remote git providers, such as GitHub.
+Git access tokens are important for Dataform projects linked to remote git providers, such as GitHub. They provide authentication for accessing and controlling the git project.
 
-Acess tokens can be updated on the **project settings page**, within the app.
+## User authentication 🙋‍♂️🙋‍♀️
 
-## User authentication
+User access tokens are used to authenticate git access of Dataform projects during project manipulation.
 
-Access tokens for **users** must have both **read and write access** to your remote repository.
+Without a valid access token, users will be unable to do things such as create branches.
 
-Access tokens for the **scheduler** must have **read** access to your remote repository.
+They must be configured to have both **read and write access** to your remote repository.
+
+They can be added or removed on the **user settings page**.
+
+## Scheduler authentication ⏱
+
+Scheduler access tokens are used to authenticate git access of Dataform projects when running automated schedules.
+
+Without a valid access token, the scheduler will not be able to read the contents of the project and perform the run.
+
+They must have **read** access to your remote repository.
+
+They can be updated on the **project settings page**, within the app.
+
+For smaller projects (e.g. 5 or fewer people), it's ok to use a user's personal access token to authenticate the scheduler, but for larger teams a **bot account** is recommended.
+
+### Bot accounts
+
+Bot accounts are generally regular user accounts, but used purely to run automated (or triggered) tasks.
+
+Some of the advantages of using a bot account include:
```

Comment:
Rephrase the reason for machine user from here: https://github.com/dataform-co/dataform/pull/620/files